js/wp-document-revisions.coffee2

```diff
@@ -20,6 +19,8 @@
       @cookieFalse = bind(@cookieFalse, this)
       @cookieTrue = bind(@cookieTrue, this)
       @cookieDelete = bind(@cookieDelete, this)
+      @buildContent = bind(@buildContent, this)
```

Comment:
It's probably time to convert this to TypeScript, but that's for a different PR 😄